### ISSUE TITLE
Typescript compilation error (Angular 8.2, Typescript 3.5)

### ISSUE THREAD

**Issue by spierala** [taye/interact.js#822]
I am using interactjs 1.9.17 in an angular project (angular 8.2 and typescript 3.5). 

Sadly the project does not compile due to Typescript errors related to interactjs:

```
ERROR in ../../node_modules/@interactjs/types/index.d.ts:679:17 - error TS1005: '=' expected.

679     import type { Options } from "@interactjs/core/defaultOptions";
                    ~
../../node_modules/@interactjs/types/index.d.ts:679:34 - error TS1005: ';' expected.

679     import type { Options } from "@interactjs/core/defaultOptions";
```

The import of interactjs looks like this:
`import * as interact from 'interactjs';`

Previously I was using "interactjs": "1.3.x" and there were no issues.

**Comment by rodrigo-fonseca** [comment]
"Previously I was using "interactjs": "1.3.x" and there were no issues." I need to downgrade to this version because it was the only one that worked :(

**Comment by fionnbharra** [comment]
Type imports were [introduced with typescript 3.8](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-exports). Is there a way to make these backwards compatible?

**Comment by taye** [comment]
I thought they would be backwards compatible. I don't see why the declaration files should leave the `type` in there when they can't export values 💁.

I'll try to avoid using them, but while I'm here I'd like to express my distaste for Angular 😛.

**Comment by taye** [comment]
I've released `v1.9.18` which hopefully resolves this issue.

**Comment by spierala** [comment]
Looks good! Thank you!
I am using this import:

`import interact from 'interactjs';`
